compiler/luci/export/src/CircleTensorExporter.cpp

```diff
@@ -132,6 +132,7 @@ class MultiOutputDetector final : public luci::CircleNodeMutableVisitor<bool>
   {
     auto outs = loco::succs(node);
     assert(outs.size() == count);
+    (void)count; // for unused variable error in release build
```

Comment:
I saw the full draft codes, but I couldn't find why `count` is needed :(
Will this variable be used for later?
If not, how about following code?
```cpp
void store_outputs(luci::CircleNode *node, uint32_t) // Remove count
```